@xen-orchestra/lite/src/App/TabConsole.tsx

```diff
@@ -0,0 +1,44 @@
+import React from 'react'
+import { withState } from 'reaclette'
+
+import Button from '../components/Button'
+import Console from '../components/Console'
+
+interface ParentState {}
+
+interface State {
+  ctrlAltDel: () => void
+}
+
+interface Props {
+  vmId: string
+}
+
+interface ParentEffects {}
+
+interface Effects {
+  setCtrlAltDel: (fn: () => void) => void
```

Comment:
You can use type extraction: `(sendCtrlAltDel: State['sendCtrlAltDel']) => void`